webui/react/src/components/kit/Facepile.tsx

```diff
@@ -0,0 +1,111 @@
+import { MinusOutlined, PlusOutlined } from '@ant-design/icons';
+import React, { useMemo, useState } from 'react';
+
+import SelectFilter from 'components/SelectFilter';
+import { DetailedUser } from 'types';
+
+import Button from './Button';
+import css from './Facepile.module.scss';
+import UserAvatar from './UserAvatar';
+
+export interface Props {
```

Comment:
This component doesn't provide any way to hook into the "add user" functionality.